tests/data/generator/two_party_negotiation/message.py

```diff
@@ -0,0 +1,91 @@
+"""This module contains two_party_negotiation's message definition."""
+
+from typing import cast, List
+
+from aea.protocols.base import Message
+
+
+class DataModel:
+    """This class represents a DataModel."""
+
+    def __init__(self):
+        """Initialise a DataModel."""
+        raise NotImplementedError
+
+    def __eq__(self, other):
+        """Compare two instances of this class."""
+        if type(other) is type(self):
+            raise NotImplementedError
+        else:
+            return False
+
+
+class TwoPartyNegotiationMessage(Message):
+    """A protocol for negotiation over a fixed set of resources involving two parties."""
+
+    def __init__(self, message_id: int, target: int, performative: str, contents: List, **kwargs):
+        """Initialise."""
+        super().__init__(message_id=message_id, target=target, performative=performative, contents=contents, **kwargs)
+
+        self.speech_acts = {
+            'cfp': {
+                'query', DataModel},
```

Comment:
Can we have the `},` in the next line properly indented?

Reply:
Done